codegen/src/test/resources/software/amazon/awssdk/codegen/poet/authscheme/auth-scheme-params.java

```diff
@@ -0,0 +1,28 @@
+package software.amazon.awssdk.services.query.authscheme;
+
+import java.util.Optional;
+import software.amazon.awssdk.annotations.Generated;
+import software.amazon.awssdk.annotations.SdkPublicApi;
+import software.amazon.awssdk.services.query.authscheme.internal.DefaultQueryAuthSchemeParams;
+
+/**
+ * The parameters object used to resolve the auth schemes for the Query service.
+ */
+@Generated("software.amazon.awssdk:codegen")
+@SdkPublicApi
+public interface QueryAuthSchemeParams {
+
+    static Builder builder() {
+        return DefaultQueryAuthSchemeParams.builder();
+    }
+
+    String operation();
```

Comment:
Is it possible to add javadocs for these methods and the builder methods?

Reply:
Oh missed some. Thanks for catching. Fixed.